### DISCUSSION TITLE
Content warning bug I guess?

### DISCUSSION THREAD

**Discussion by sbug0** [elk-zone/elk#1801] [General]
Scrolling through the time line I saw this.
It's a content warning post but Elk is not showing it as such.
The images show how Elk show the post compared to Mastodon web.

<img width="609" alt="Elk" src="https://user-images.githubusercontent.com/122732288/219947513-b980bfde-cbf6-4cdf-940c-bb30f1f02aab.png">
<img width="582" alt="Mastodon" src="https://user-images.githubusercontent.com/122732288/219947519-4d59827e-715f-486e-bd6c-2cd3e928b83b.png">

**Comment by ayo-run** [comment]
Yeah looks like a bug indeed. Hi @sbug0, do you still see this?

**Reply by sbug0** [reply]
Hi. Yes I'm still seeing this.

Link to the post:
https://infosec.exchange/@catsalad/109890930166069976

**Reply by ayo-run** [reply]
Created an issue #1846 -- I'll work on it if no one picks it up before me

**Comment by ayo-run** [comment]
This should be fixed by #1864
